charts/openshift-metering/templates/hive/hive-server-statefulset.yaml

```diff
@@ -124,6 +124,33 @@ spec:
 {{- end }}
         resources:
 {{ toYaml .Values.hive.spec.server.resources | indent 10 }}
+{{- if .Values.hive.spec.server.config.tls.enabled }}
+      - name: ghostunnel-client
+        image: {{ .Values.__ghostunnel.image.repository }}:{{ .Values.__ghostunnel.image.tag }}
+        imagePullPolicy: {{ .Values.__ghostunnel.image.pullPolicy }}
+        env:
+        - name: HIVE_SERVER_CA_CRTFILE
+          value: /opt/hive-server/tls/ca.crt
```

Comment:
Let's use `/opt/hive/tls`

Reply:
I went with hive-server because the metastore is also opt/hive/tls, but I'll change it cause it's a different pod 👍